homeassistant/components/songpal/media_player.py

```diff
@@ -290,6 +333,27 @@ def source_list(self):
         """Return list of available sources."""
         return [src.title for src in self._sources.values()]
 
+    async def async_select_sound_mode(self, sound_mode: str) -> None:
+        """Select sound mode."""
+        for out in self._sound_modes.values():
+            if out.title == sound_mode:
+                await self._dev.set_sound_settings("soundField", out.value)
+                return
```

Comment:
`out` is not a good name for the iterator variable, maybe name it `mode` or just name it `val`:
```suggestion
        for mode in self._sound_modes.values():
            if mode.title == sound_mode:
                await self._dev.set_sound_settings("soundField", mode.value)
                return
```